### PR TITLE
[WIP] remove arc.js

### PR DESCRIPTION
**Pull request by jellegerbrandy**

### CODE REVIEW

File: src/reducers/notifications.ts [hunk 1]
```diff
@@ -163,23 +163,23 @@ function elipsis(str: string, n: number) {
...
-  'GenesisProtocol.redeemDaoBounty': (proposalTitle, {proposalId}: Arc.RedeemConfig) =>
+  'GenesisProtocol.redeemDaoBounty': (proposalTitle, {proposalId}) =>
     `Redeeming bounty rewards for "${elipsis(proposalTitle, 22)}"`,
-  'ContributionReward.proposeContributionReward': (proposalTitle, {}: Arc.ProposeContributionRewardParams) =>
+  'ContributionReward.proposeContributionReward': (proposalTitle, {}) =>
     `Creating proposal ${elipsis(proposalTitle, 22)}`,
-  'ContributionReward.redeemContributionReward': (proposalTitle, {proposalId}: Arc.ContributionRewardRedeemParams) =>
+  'ContributionReward.redeemContributionReward': (proposalTitle, {proposalId}) =>
     `Redeeming contribution reward for "${elipsis(proposalTitle, 22)}"`,
-  'DAO.new': (proposalTitle, {}: Arc.NewDaoConfig) =>
```
Comment (tibetsprague) [inline src/reducers/notifications.ts]: didnt we get rid of DAO creation?

File: src/reducers/notifications.ts [hunk 2]
```diff
@@ -163,23 +163,23 @@ function elipsis(str: string, n: number) {
  * A map of messages to show for each type of action.
  */
 const messages: {[key: string]: (proposalTitle: string | undefined, options: any) => string} = {
-  'GenesisProtocol.vote': (proposalTitle, {vote, proposalId}: Arc.VoteOptions) =>
```
Comment (tibetsprague) [inline src/reducers/notifications.ts]: we probably want to replace all these types, they are useful and maybe necessary to have in some places

File: src/components/DaoList/DaoContainer.tsx [hunk 3]
```diff
@@ -1,7 +1,7 @@
 import * as React from "react";
 import { Link } from "react-router-dom";
 import Subscribe, { IObservableState } from "components/Shared/Subscribe"
-import { arc } from "arc"
+import { getArc } from 'arc'
```
Comment (tibetsprague) [inline src/components/DaoList/DaoContainer.tsx]: i thought we were changing everything to double quotes abut now you seem to be changing everything to single quotes

File: src/components/ViewDao/DaoNav.tsx [hunk 4]
```diff
@@ -0,0 +1,52 @@
+import Tooltip from 'rc-tooltip';
```
Comment (tibetsprague) [inline src/components/ViewDao/DaoNav.tsx]: DaoNav has been removed, everything is now in DaoSidebar

File: src/components/DaoList/DaoContainer.tsx (hunk 3 above)
Comment (jellegerbrandy) [reply src/components/DaoList/DaoContainer.tsx]: My muscle memory types singe quotes. We need to enable the linter rule.  I've made an issue: https://github.com/daostack/alchemy/issues/295

**Pull request merged**